securedrop/tests/test_db.py

```diff
@@ -78,3 +78,10 @@ def test_journalist_string_representation(self):
     def test_source_string_representation(self):
         test_source, _ = db_helper.init_source()
         test_source.__repr__()
+
+    def test_throttle_login(self):
+        journalist, _ = db_helper.init_journalist()
+        for _ in range(Journalist._MAX_LOGIN_ATTEMPTS_PER_PERIOD):
+            Journalist.throttle_login(journalist)
+        with self.assertRaises(LoginThrottledException):
```

Comment:
You're not wrong. I would use `assertRaisesRegexp` as a defensive programming measure in case`LoginThrottledException` does find a second use in the future. The reason not to wait until then is just to be careful, so you don't forget if/ when `LoginThrottledException` finds a second use.

I realize in this specific example it's highly unlikely, but I just have a general mindset of defensive programming and making test assertions as comprehensive as practical. I approved this PR because it definitely is good as it was merged, and making this change was not worth burning another review cycle on.